cmds/core/mv/mv.go

```diff
@@ -31,16 +31,16 @@ var (
 	noClobber = flag.Bool("n", false, "do not overwrite an existing file")
 )
 
-func moveFile(source string, dest string) error {
-	if *noClobber {
+func moveFile(update, noClobber bool, source string, dest string) error {
```

Comment:
it's a very minor thing, but I feel this might follow the ordering of other Go functions such as io.Copy:
moveFile(dest, source string, update, noClobber bool)
it just looks a little nicer to me?

Reply:
Sure, updated!